packages/core/src/utils/strings.ts

```diff
@@ -0,0 +1,10 @@
+export function kebabCase(string: string) {
+  return string
+    .replace(/([a-z])([A-Z])/g, '$1-$2')
+    .replace(/[\s_]+/g, '-')
+    .toLowerCase()
```

Comment:
This function doesn't return exactly the same results as `lodash`'s version, but I'm not sure how true to the original we need to be. The most obvious difference is that hyphens can end up leading or trailing characters -- is it worth an enhancement to trim that?
<img width="668" alt="Screenshot 2024-01-24 at 7 15 24 AM" src="https://github.com/priceline/design-system/assets/5330173/b033bfdc-5bc8-4247-acdd-33052cc73064">
<img width="210" alt="Screenshot 2024-01-24 at 7 16 10 AM" src="https://github.com/priceline/design-system/assets/5330173/d13c5ba5-8262-44ce-9278-250b0b7ed0be">

Reply:
good catch! it's a naive copy from the internet, first goal was to get it building and measure the bundle change